### ISSUE TITLE
the arbitrator should be able to change the beneficiary

### ISSUE THREAD

**Issue by jellegerbrandy** [hats-finance/hats-contracts#479]
This means changed the `approveClaim` function and adding a beneficiary argument:

```` 
function approveClaim(bytes32 _claimId, address _beneficiary, uint16 _bountyPercentage) {
...
            if (_claim.arbitratorCanChangeBeneficiary) {
                _claim.beneficiary = _beneficiary;
            }
...
}
````

We will also need a new vault seeing `arbitratorCanChangeBeneficiary` which uses exactly the same logic as `arbitratorCanChangeBounty` (i.e. by setting it by default to NO)

**Comment by shayzluf** [comment]
@ben-kaufman is this spec ok with you?

**Comment by jellegerbrandy** [comment]
resolved in #487
